### ISSUE TITLE
App doesn't recognize playback from Browser

### ISSUE THREAD

**Issue by Alistair1231** [advplyr/audiobookshelf-app#984]
### Steps to reproduce
I am on v2.6.0 Docker / 0.9.68-beta and playback sync only works from Android -> Browser. Any time I listen in the Browser, and then switch to Android, the progress doesn't get synced. It doesn't matter if the Audiobook is downloaded or not.
The only workaround is to kill the app before I resume on Android, and then restart it. Then it gets the correct position, if I forget to kill the app, the sync progress gets lost and I need to manually go to the position I look up in my Browser Listening Session history.

Here are some screenshots and logs:

---------------------
## Example 1 - History screen on Android and Browser after I paused in my Browser

Here I paused at 16:59 on Android and listened until 17:55 on my Browser. Then I switched to Android and resumed. It kept playing from 16:59.
This is what the Listening Sessions view in the Browser looked like at that point:
<img src="https://github.com/advplyr/audiobookshelf-app/assets/26165288/4ad903df-6f93-44a9-878f-f2f9c781d28f">

This is what the History in Android showed me:

<img width=30% src="https://github.com/advplyr/audiobookshelf-app/assets/26165288/2fe15a9e-c270-4f01-ba57-194f3fa2dc4f">

<details ><summary>

### Logs

</summary>

```
abs-container  | [2023-12-12 15:44:33.082] DEBUG: [ApiCacheManager] Object.afterBulkUpdate: Clearing cache (ApiCacheManager.js:21)
abs-container  | [2023-12-12 15:44:43.124] DEBUG: [PlaybackSessionManager] syncSession "0aef36a6-6c05-4606-b026-0cb783080059" (Device: Windows 10 / Firefox) | Total Time Listened: 3840 (PlaybackSessionManager.js:276)
abs-container  | [2023-12-12 15:44:43.130] DEBUG: [ApiCacheManager] Object.afterBulkUpdate: Clearing cache (ApiCacheManager.js:21)
abs-container  | [2023-12-12 15:44:53.315] DEBUG: [PlaybackSessionManager] syncSession "0aef36a6-6c05-4606-b026-0cb783080059" (Device: Windows 10 / Firefox) | Total Time Listened: 3850 (PlaybackSessionManager.js:276)
...
abs-container  | [2023-12-12 16:26:19.075] DEBUG: [PlaybackSessionManager] syncSession "0aef36a6-6c05-4606-b026-0cb783080059" (Device: Windows 10 / Firefox) | Total Time Listened: 6310 (PlaybackSessionManager.js:276)
abs-container  | [2023-12-12 16:26:19.080] DEBUG: [ApiCacheManager] Object.afterBulkUpdate: Clearing cache (ApiCacheManager.js:21)
abs-container  | [2023-12-12 16:26:29.271] DEBUG: [PlaybackSessionManager] syncSession "0aef36a6-6c05-4606-b026-0cb783080059" (Device: Windows 10 / Firefox) | Total Time Listened: 6320 (PlaybackSessionManager.js:276)
abs-container  | [2023-12-12 16:26:29.276] DEBUG: [ApiCacheManager] Object.afterBulkUpdate: Clearing cache (ApiCacheManager.js:21)
abs-container  | [2023-12-12 16:26:39.413] DEBUG: [PlaybackSessionManager] syncSession "0aef36a6-6c05-4606-b026-0cb783080059" (Device: Windows 10 / Firefox) | Total Time Listened: 6330 (PlaybackSessionManager.js:276)
abs-container  | [2023-12-12 16:26:39.418] DEBUG: [ApiCacheManager] Object.afterBulkUpdate: Clearing cache (ApiCacheManager.js:21)
abs-container  | [2023-12-12 16:29:28.224] INFO: [SocketAuthority] Socket Connected kBNnf-6r95PhECmWAADm
abs-container  | [2023-12-12 16:29:28.362] DEBUG: [SocketAuthority] User Online root (SocketAuthority.js:191)
abs-container  | [2023-12-12 16:29:42.708] DEBUG: [PlaybackSessionManager] syncSession "069c98d6-e312-4b50-8f18-5fa8f6756886" (Device: Pixel 8 SDK 34 / v14) | Total Time Listened: 244 (PlaybackSessionManager.js:276)
abs-container  | [2023-12-12 16:29:42.713] DEBUG: [ApiCacheManager] Object.afterBulkUpdate: Clearing cache (ApiCacheManager.js:21)
abs-container  | [2023-12-12 16:29:57.479] DEBUG: [PlaybackSessionManager] syncSession "069c98d6-e312-4b50-8f18-5fa8f6756886" (Device: Pixel 8 SDK 34 / v14) | Total Time Listened: 258 (PlaybackSessionManager.js:276)
abs-container  | [2023-12-12 16:29:57.484] DEBUG: [ApiCacheManager] Object.afterBulkUpdate: Clearing cache (ApiCacheManager.js:21)
abs-container  | [2023-12-12 16:30:00.942] DEBUG: [PlaybackSessionManager] syncSession "069c98d6-e312-4b50-8f18-5fa8f6756886" (Device: Pixel 8 SDK 34 / v14) | Total Time Listened: 261 (PlaybackSessionManager.js:276)
abs-container  | [2023-12-12 16:30:00.947] DEBUG: [ApiCacheManager] Object.afterBulkUpdate: Clearing cache (ApiCacheManager.js:21)
abs-container  | [2023-12-12 16:33:24.871] DEBUG: [SocketAuthority] User Offline root (SocketAuthority.js:118)
abs-container  | [2023-12-12 16:33:24.874] INFO: [SocketAuthority] Socket kBNnf-6r95PhECmWAADm disconnected from client "root" after 236650ms (Reason: transport close)
abs-container  | [2023-12-12 16:33:27.179] DEBUG: [ApiCacheManager] count: 0 size: 0 (ApiCacheManager.js:29)
abs-container  | [2023-12-12 16:33:27.183] DEBUG: [ApiCacheManager] Cache miss: {"user":"root","url":"/libraries"} (ApiCacheManager.js:40)
abs-container  | [2023-12-12 16:33:27.261] INFO: [SocketAuthority] Socket Connected 6XYKnGKDuDAgEdlmAADo
abs-container  | [2023-12-12 16:33:27.343] DEBUG: [ApiCacheManager] count: 1 size: 1504 (ApiCacheManager.js:29)
abs-container  | [2023-12-12 16:33:27.360] DEBUG: [SocketAuthority] User Online root (SocketAuthority.js:191)
abs-container  | [2023-12-12 16:33:27.389] DEBUG: Loaded filterdata in 0.03s (libraryFilters.js:517)
abs-container  | [2023-12-12 16:33:27.390] DEBUG: [ApiCacheManager] Cache miss: {"user":"root","url":"/libraries/74541e71-3808-4570-a1b9-a7fd2c9b8395?include=filterdata"} (ApiCacheManager.js:40)
abs-container  | [2023-12-12 16:33:27.587] DEBUG: [ApiCacheManager] count: 2 size: 13766 (ApiCacheManager.js:29)
abs-container  | [2023-12-12 16:33:27.810] DEBUG: Loaded 7 personalized shelves in 0.22s (LibraryItem.js:667)
abs-container  | [2023-12-12 16:33:27.811] DEBUG: [ApiCacheManager] Cache miss: {"user":"root","url":"/libraries/74541e71-3808-4570-a1b9-a7fd2c9b8395/personalized?minified=1&include=rssfeed,numEpisodesIncomplete"} (ApiCacheManager.js:40)
abs-container  | [2023-12-12 16:33:27.812] DEBUG: [ApiCacheManager] Caching with 1800000 ms TTL (ApiCacheManager.js:43)
abs-container  | [2023-12-12 16:35:02.433] DEBUG: [SocketAuthority] User Offline root (SocketAuthority.js:118)
abs-container  | [2023-12-12 16:35:02.435] INFO: [SocketAuthority] Socket 6XYKnGKDuDAgEdlmAADo disconnected from client "root" after 95174ms (Reason: ping timeout)
abs-container  | [2023-12-12 16:35:06.039] DEBUG: [SocketAuthority] User Offline root (SocketAuthority.js:118)
abs-container  | [2023-12-12 16:35:06.040] INFO: [SocketAuthority] Socket QF2SEIqvy0eG48vYAADS disconnected from client "root" after 8286414ms (Reason: transport close)
abs-container  | [2023-12-12 16:35:17.733] INFO: [SocketAuthority] Socket Connected jDWqXXws4lNZuJUOAADq
abs-container  | [2023-12-12 16:35:17.844] DEBUG: [SocketAuthority] User Online root (SocketAuthority.js:191)
```

</details>

---------------------
## Example 2 - History screen on Android after I killed and restarted the App

In this example I listened from 18:07 for 2min, until 18:09 in my Browser. Then  and then after I saw that the progress did not sync, I let it play for < 2sec to make sure it really didn't go to the correct position, then I killed and restarted the app. You can see, that it now realizes I was at 18:09:49, even though before the restart it kept playing 18:07:28. 

<img width=30% src="https://github.com/advplyr/audiobookshelf-app/assets/26165288/2c0d059b-8a43-4cba-812a-a4e65854f073">

<details ><summary>

### Logs

</summary>

```
abs-container  | [2023-12-12 19:54:08.917] INFO: [SocketAuthority] Socket Connected nB4hpWNlwD1CQrrNAAD8
abs-container  | [2023-12-12 19:54:08.999] DEBUG: [SocketAuthority] User Online root (SocketAuthority.js:191)
abs-container  | [2023-12-12 19:55:36.412] DEBUG: [ApiCacheManager] Object.afterBulkUpdate: Clearing cache (ApiCacheManager.js:21)
abs-container  | [2023-12-12 19:55:36.413] DEBUG: [PlaybackSessionManager] closeSession "0aef36a6-6c05-4606-b026-0cb783080059" (PlaybackSessionManager.js:306)
abs-container  | [2023-12-12 19:55:36.414] DEBUG: [PlaybackSessionManager] Removed session "0aef36a6-6c05-4606-b026-0cb783080059" (PlaybackSessionManager.js:330)
abs-container  | [2023-12-12 19:57:25.312] DEBUG: [SocketAuthority] User Offline root (SocketAuthority.js:118)
abs-container  | [2023-12-12 19:57:25.315] INFO: [SocketAuthority] Socket nB4hpWNlwD1CQrrNAAD8 disconnected from client "root" after 196398ms (Reason: ping timeout)
abs-container  | [2023-12-12 19:59:12.208] DEBUG: [ApiCacheManager] count: 0 size: 0 (ApiCacheManager.js:29)
abs-container  | [2023-12-12 19:59:12.211] DEBUG: [ApiCacheManager] Cache miss: {"user":"root","url":"/libraries"} (ApiCacheManager.js:40)
abs-container  | [2023-12-12 19:59:12.314] INFO: [SocketAuthority] Socket Connected XidpqPi-j_I6D3T5AAD-
abs-container  | [2023-12-12 19:59:12.364] DEBUG: [ApiCacheManager] count: 1 size: 1504 (ApiCacheManager.js:29)
abs-container  | [2023-12-12 19:59:12.367] DEBUG: [ApiCacheManager] Cache miss: {"user":"root","url":"/libraries/74541e71-3808-4570-a1b9-a7fd2c9b8395?include=filterdata"} (ApiCacheManager.js:40)
abs-container  | [2023-12-12 19:59:12.382] DEBUG: [SocketAuthority] User Online root (SocketAuthority.js:191)
abs-container  | [2023-12-12 19:59:12.577] DEBUG: [ApiCacheManager] count: 2 size: 13766 (ApiCacheManager.js:29)
abs-container  | [2023-12-12 19:59:12.778] DEBUG: Loaded 7 personalized shelves in 0.20s (LibraryItem.js:667)
abs-container  | [2023-12-12 19:59:12.779] DEBUG: [ApiCacheManager] Cache miss: {"user":"root","url":"/libraries/74541e71-3808-4570-a1b9-a7fd2c9b8395/personalized?minified=1&include=rssfeed,numEpisodesIncomplete"} (ApiCacheManager.js:40)
abs-container  | [2023-12-12 19:59:12.780] DEBUG: [ApiCacheManager] Caching with 1800000 ms TTL (ApiCacheManager.js:43)
abs-container  | [2023-12-12 20:00:15.945] DEBUG: [ApiCacheManager] count: 3 size: 240859 (ApiCacheManager.js:29)
abs-container  | [2023-12-12 20:00:15.945] DEBUG: [ApiCacheManager] Cache hit: {"user":"root","url":"/libraries/74541e71-3808-4570-a1b9-a7fd2c9b8395?include=filterdata"} (ApiCacheManager.js:32)
abs-container  | [2023-12-12 20:00:16.234] DEBUG: [ApiCacheManager] count: 3 size: 240859 (ApiCacheManager.js:29)
abs-container  | [2023-12-12 20:00:16.424] DEBUG: Loaded 7 personalized shelves in 0.19s (LibraryItem.js:667)
abs-container  | [2023-12-12 20:00:16.425] DEBUG: [ApiCacheManager] Cache miss: {"user":"root","url":"/libraries/74541e71-3808-4570-a1b9-a7fd2c9b8395/personalized?include=rssfeed,numEpisodesIncomplete"} (ApiCacheManager.js:40)
abs-container  | [2023-12-12 20:00:16.425] DEBUG: [ApiCacheManager] Caching with 1800000 ms TTL (ApiCacheManager.js:43)
abs-container  | [2023-12-12 20:00:21.991] DEBUG: [SocketAuthority] User Offline root (SocketAuthority.js:118)
abs-container  | [2023-12-12 20:00:21.992] INFO: [SocketAuthority] Socket XidpqPi-j_I6D3T5AAD- disconnected from client "root" after 69678ms (Reason: transport close)
abs-container  | [2023-12-12 20:00:24.514] DEBUG: [ApiCacheManager] count: 4 size: 468730 (ApiCacheManager.js:29)
abs-container  | [2023-12-12 20:00:24.514] DEBUG: [ApiCacheManager] Cache hit: {"user":"root","url":"/libraries"} (ApiCacheManager.js:32)
abs-container  | [2023-12-12 20:00:24.649] INFO: [SocketAuthority] Socket Connected LGAKRVQ7BtM7VQ36AAEA
abs-container  | [2023-12-12 20:00:24.649] DEBUG: [ApiCacheManager] count: 4 size: 468730 (ApiCacheManager.js:29)
abs-container  | [2023-12-12 20:00:24.650] DEBUG: [ApiCacheManager] Cache hit: {"user":"root","url":"/libraries/74541e71-3808-4570-a1b9-a7fd2c9b8395?include=filterdata"} (ApiCacheManager.js:32)
abs-container  | [2023-12-12 20:00:24.757] DEBUG: [SocketAuthority] User Online root (SocketAuthority.js:191)
abs-container  | [2023-12-12 20:00:24.780] DEBUG: [ApiCacheManager] count: 4 size: 468730 (ApiCacheManager.js:29)
abs-container  | [2023-12-12 20:00:24.781] DEBUG: [ApiCacheManager] Cache hit: {"user":"root","url":"/libraries/74541e71-3808-4570-a1b9-a7fd2c9b8395/personalized?minified=1&include=rssfeed,numEpisodesIncomplete"} (ApiCacheManager.js:32)
abs-container  | [2023-12-12 20:00:28.166] DEBUG: [ApiCacheManager] Object.afterBulkUpdate: Clearing cache (ApiCacheManager.js:21)
abs-container  | [2023-12-12 20:00:28.166] DEBUG: [PlaybackSessionManager] startSessionRequest for device Pixel 8 SDK 34 / v14 (PlaybackSessionManager.js:64)
abs-container  | [2023-12-12 20:00:28.166] INFO: [PlaybackSessionManager] startSession: Closing open session "He Who Fights with Monsters 9" for user "root" (Device: Pixel 8 SDK 34 / v14)
abs-container  | [2023-12-12 20:00:28.169] DEBUG: [ApiCacheManager] Object.afterBulkUpdate: Clearing cache (ApiCacheManager.js:21)
abs-container  | [2023-12-12 20:00:28.169] DEBUG: [PlaybackSessionManager] closeSession "069c98d6-e312-4b50-8f18-5fa8f6756886" (PlaybackSessionManager.js:306)
abs-container  | [2023-12-12 20:00:28.170] DEBUG: [PlaybackSessionManager] Removed session "069c98d6-e312-4b50-8f18-5fa8f6756886" (PlaybackSessionManager.js:330)
abs-container  | [2023-12-12 20:00:28.170] DEBUG: [PlaybackSessionManager] "root" starting direct play session for item "8bd0eae2-adef-4732-a41e-db91f07352cb" with id 86dbab6d-7745-4aa2-8c6e-f8ad542bd9b0 (Device: Pixel 8 SDK 34 / v14) (PlaybackSessionManager.js:240)
abs-container  | [2023-12-12 20:00:29.053] INFO: Received ping
abs-container  | [2023-12-12 20:00:29.490] DEBUG: [ApiCacheManager] count: 0 size: 0 (ApiCacheManager.js:29)
abs-container  | [2023-12-12 20:00:29.493] DEBUG: [ApiCacheManager] Cache miss: {"user":"root","url":"/libraries"} (ApiCacheManager.js:40)
abs-container  | [2023-12-12 20:00:35.065] DEBUG: [PlaybackSessionManager] syncSession "86dbab6d-7745-4aa2-8c6e-f8ad542bd9b0" (Device: Pixel 8 SDK 34 / v14) | Total Time Listened: 2 (PlaybackSessionManager.js:276)
abs-container  | [2023-12-12 20:00:35.069] DEBUG: [ApiCacheManager] playbackSession.afterCreate: Clearing cache (ApiCacheManager.js:21)
abs-container  | [2023-12-12 20:00:37.539] DEBUG: [PlaybackSessionManager] startSessionRequest for device Windows 10 / Firefox (PlaybackSessionManager.js:64)
abs-container  | [2023-12-12 20:00:37.540] DEBUG: [PlaybackSessionManager] "root" starting direct play session for item "8bd0eae2-adef-4732-a41e-db91f07352cb" with id 8125e93d-59f9-424f-a966-e305d604a309 (Device: Windows 10 / Firefox) (PlaybackSessionManager.js:240)
abs-container  | [2023-12-12 20:00:58.380] DEBUG: [PlaybackSessionManager] syncSession "8125e93d-59f9-424f-a966-e305d604a309" (Device: Windows 10 / Firefox) | Total Time Listened: 20 (PlaybackSessionManager.js:276)
abs-container  | [2023-12-12 20:00:58.398] DEBUG: [ApiCacheManager] playbackSession.afterCreate: Clearing cache (ApiCacheManager.js:21)
abs-container  | [2023-12-12 20:01:08.452] DEBUG: [PlaybackSessionManager] syncSession "8125e93d-59f9-424f-a966-e305d604a309" (Device: Windows 10 / Firefox) | Total Time Listened: 30 (PlaybackSessionManager.js:276)
...
abs-container  | [2023-12-12 20:03:09.643] DEBUG: [PlaybackSessionManager] syncSession "8125e93d-59f9-424f-a966-e305d604a309" (Device: Windows 10 / Firefox) | Total Time Listened: 150 (PlaybackSessionManager.js:276)
abs-container  | [2023-12-12 20:03:09.648] DEBUG: [ApiCacheManager] Object.afterBulkUpdate: Clearing cache (ApiCacheManager.js:21)
abs-container  | [2023-12-12 20:03:19.706] DEBUG: [PlaybackSessionManager] syncSession "8125e93d-59f9-424f-a966-e305d604a309" (Device: Windows 10 / Firefox) | Total Time Listened: 160 (PlaybackSessionManager.js:276)
abs-container  | [2023-12-12 20:03:19.711] DEBUG: [ApiCacheManager] Object.afterBulkUpdate: Clearing cache (ApiCacheManager.js:21)
abs-container  | [2023-12-12 20:03:30.603] DEBUG: [SocketAuthority] User Offline root (SocketAuthority.js:118)
abs-container  | [2023-12-12 20:03:30.604] INFO: [SocketAuthority] Socket LGAKRVQ7BtM7VQ36AAEA disconnected from client "root" after 185955ms (Reason: transport close)
abs-container  | [2023-12-12 20:03:32.843] DEBUG: [ApiCacheManager] count: 0 size: 0 (ApiCacheManager.js:29)
abs-container  | [2023-12-12 20:03:32.846] DEBUG: [ApiCacheManager] Cache miss: {"user":"root","url":"/libraries"} (ApiCacheManager.js:40)
abs-container  | [2023-12-12 20:03:33.008] INFO: [SocketAuthority] Socket Connected NBgBSLQ0ZJyN_HOwAAEC
abs-container  | [2023-12-12 20:03:33.008] DEBUG: [ApiCacheManager] count: 1 size: 1504 (ApiCacheManager.js:29)
abs-container  | [2023-12-12 20:03:33.011] DEBUG: [ApiCacheManager] Cache miss: {"user":"root","url":"/libraries/74541e71-3808-4570-a1b9-a7fd2c9b8395?include=filterdata"} (ApiCacheManager.js:40)
abs-container  | [2023-12-12 20:03:33.127] DEBUG: [SocketAuthority] User Online root (SocketAuthority.js:191)
abs-container  | [2023-12-12 20:03:33.152] DEBUG: [ApiCacheManager] count: 2 size: 13766 (ApiCacheManager.js:29)
abs-container  | [2023-12-12 20:03:33.346] DEBUG: Loaded 7 personalized shelves in 0.19s (LibraryItem.js:667)
abs-container  | [2023-12-12 20:03:33.347] DEBUG: [ApiCacheManager] Cache miss: {"user":"root","url":"/libraries/74541e71-3808-4570-a1b9-a7fd2c9b8395/personalized?minified=1&include=rssfeed,numEpisodesIncomplete"} (ApiCacheManager.js:40)
abs-container  | [2023-12-12 20:03:33.347] DEBUG: [ApiCacheManager] Caching with 1800000 ms TTL (ApiCacheManager.js:43)
abs-container  | [2023-12-12 20:03:35.969] DEBUG: [ApiCacheManager] Object.afterBulkUpdate: Clearing cache (ApiCacheManager.js:21)
abs-container  | [2023-12-12 20:03:35.969] DEBUG: [PlaybackSessionManager] startSessionRequest for device Pixel 8 SDK 34 / v14 (PlaybackSessionManager.js:64)
abs-container  | [2023-12-12 20:03:35.969] INFO: [PlaybackSessionManager] startSession: Closing open session "He Who Fights with Monsters 9" for user "root" (Device: Pixel 8 SDK 34 / v14)
abs-container  | [2023-12-12 20:03:35.972] DEBUG: [ApiCacheManager] Object.afterBulkUpdate: Clearing cache (ApiCacheManager.js:21)
abs-container  | [2023-12-12 20:03:35.972] DEBUG: [PlaybackSessionManager] closeSession "86dbab6d-7745-4aa2-8c6e-f8ad542bd9b0" (PlaybackSessionManager.js:306)
abs-container  | [2023-12-12 20:03:35.973] DEBUG: [PlaybackSessionManager] Removed session "86dbab6d-7745-4aa2-8c6e-f8ad542bd9b0" (PlaybackSessionManager.js:330)
abs-container  | [2023-12-12 20:03:35.973] DEBUG: [PlaybackSessionManager] "root" starting direct play session for item "8bd0eae2-adef-4732-a41e-db91f07352cb" with id 1db7bf88-a1f3-4472-9a65-207ddbbd5941 (Device: Pixel 8 SDK 34 / v14) (PlaybackSessionManager.js:240)
abs-container  | [2023-12-12 20:03:36.772] INFO: Received ping
abs-container  | [2023-12-12 20:03:37.102] DEBUG: [ApiCacheManager] count: 0 size: 0 (ApiCacheManager.js:29)
abs-container  | [2023-12-12 20:03:37.104] DEBUG: [ApiCacheManager] Cache miss: {"user":"root","url":"/libraries"} (ApiCacheManager.js:40)
abs-container  | [2023-12-12 20:03:42.365] DEBUG: [PlaybackSessionManager] syncSession "1db7bf88-a1f3-4472-9a65-207ddbbd5941" (Device: Pixel 8 SDK 34 / v14) | Total Time Listened: 1 (PlaybackSessionManager.js:276)
abs-container  | [2023-12-12 20:03:42.370] DEBUG: [ApiCacheManager] playbackSession.afterCreate: Clearing cache (ApiCacheManager.js:21)

```

</details>

1. Listen on Android
2. Pause playback and switch to Browser
3. Pause Browser and switch back to Android

### Expected behaviour
- Playback should resume at the position the Browser is at, and Browser session should be visible in History on Android.

### Actual behaviour
- Android does not realize any other playback happened and browser session does not appear in the History. 

### Environment data

Audiobookshelf Version:

- [X] Android App?
- [ ] iOS App?

#### Android Issue
Android version: 0.9.68-beta

Device model: Pixel 8 (A14)

Stock or customized system: Stock

**Comment by advplyr** [comment]
I was able to reproduce this. This is specific to the audio player staying open in Android.
There are actually 2 issues here and downloaded books hit both of them.

1. The audio player trackbar progress does not update when the browser is playing (the browser trackbar does update in realtime)
2. For downloaded books if the audio track is open in Android then you won't see "Sync"'s in the item history.

For the second one you can test it by closing the audio player in Android and going to the history page for that book. Then play in the browser and you should start seeing Sync's come in after 30 seconds.

**Comment by Alistair1231** [comment]
Regarding the 2. Point I can confirm, that It does sync, when I start Browser, while the App is closed, and only after the playback begins, open the App. In this case, it even keeps updating and showing Sync +1, +2, etc
After playing and pausing on Android it does stop Syncing and only starts syncing again after restarting the app or using the "close player" menu item.
So it seems the problem is, that no updates can happen while playback is in progress, even if it is paused.

**Comment by advplyr** [comment]
Fixed in [0.9.69-beta](https://github.com/advplyr/audiobookshelf-app/releases/tag/v0.9.69-beta)
